doc/guides/technical-values.md

```diff
@@ -20,6 +20,8 @@ with Node.js. Some key elements of this include:
 * Great documentation
 * Bundling friction-reducing APIs and components, even though
   they could be provided externally
+* Maximizing compatibility and interoperability with browsers and other
```

Comment:
Discussion from the next-10 meeting was that those in the meeting were more comfortable with:

`Compatibility and interoperability with browsers and other JavaScript environments where it fits in the Node.js context`

Reply:
We ran out of time so, people from the team will jump in here directly to continue the discussion.

Reply:
I’m not sure what “where it fits in the Node.js context” means. Couldn’t you say that about any of the values in this document?

I’m happy to discuss any concerns people have.